### ISSUE TITLE
Invalid "some routes are not used at all" messages

### ISSUE THREAD

**Issue by davidpnewton** [Zaczero/osm-relatify#15]
Created relation 16017925 to test the software.

Added ways 1146049452 and 1146049452 to test the software.

Loaded the relation into the software and then tried to add further ways to the relation. In all cases the message "some routes are not used at all" came up. It did not matter whether I started with way 1146049451 or way 1076459329, the same invalid message resulted in both cases.

![image](https://github.com/Zaczero/osm-relatify/assets/74616419/d5ab7455-8e4b-41be-a920-dca72a00c442)

![image](https://github.com/Zaczero/osm-relatify/assets/74616419/aea2b594-0a68-4b99-87e7-581e0f2fafb5)

![image](https://github.com/Zaczero/osm-relatify/assets/74616419/433da79e-893f-4b07-ba5e-f244794a6821)

The software is clearly not detecting the direction of the route properly in this scenario.

**Comment by Zaczero** [comment]
In both of the screenshots you provided, there are 3 selected ways, with an extra way on the left, and an extra way on the right. If the start/stop points are not relocated, such routes would be considered impossible. You should either relocate the start/stop point (right-click on a way) or remove the unused way.

**Comment by Zaczero** [comment]
I believe the issue lies in the fact that the start/stop button relocation menu is somewhat hidden and not very intuitive. I will explore if there's anything I can do to address this!

**Comment by davidpnewton** [comment]
Right-clicking sorted the problem. So it does appear to be a UI intuition problem rather than a functionality bug. So how to make this more intuitive?

I would suggest altering the Some ways are not used error dialogue box as a possibility. The error message there at the moment seems to assume that unused ways need deselecting. Perhaps altering the language to ways not being between the start of the route and end of the route would be a possibility.  Alternatively create a separate error box for that possibility which has instructions on how to move the start and end of the route.

**Comment by Zaczero** [comment]
@davidpnewton Thank you for your feedback! User experience is very important to me. I will see what can be done about it, and I will keep your suggestions in mind. This, however, won't be fixed this month (June).

**Comment by JoeyJoJoJr** [comment]
I have been having this issue with two roundtrip bus routes, which I know (in JOSM) are correctly formatted

https://www.openstreetmap.org/relation/79492
https://www.openstreetmap.org/relation/6778795

On the mobile interface there is no possibility to right-click and move the start or end point, which are both correct regardless
